eightshift-projects/bin/db-import.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Script used to export database and images to zip.
+ * Must provide from and to parameters that are defined in setup.json file.
+ *
+ * Available commands:
+ * - php bin/db-import.php --from=staging --to=develop
+ * - php bin/db-import.php --from=production --to=develop
+ */
+
+// Define project root.
+$project_root_path = dirname( __FILE__, 2 );
+
+// Define setup.json file path.
+$setup_file = 'setup.json';
+
+// Provide mandatory parameters.
+$args = getopt(
+  null,
+  [
+    "from:",
+    "to:",
+  ]
+);
+
+// Check if mandatory parameters exists.
+$from = isset( $args['from'] ) ? $args['from'] : '';
+$to   = isset( $args['to'] ) ? $args['to'] : '';
+
+if ( empty( $from ) ) {
+  echo "--from parameter is mandatory. Please provide one url key from {$setup_file} file.",
+  die;
+}
+
+if ( empty( $to ) ) {
+  echo "--to parameter is mandatory. Please provide one url key from {$setup_file} file.",
+  die;
+}
+
+// Change execution folder.
+chdir( $project_root_path );
+
+// Check if setup exists.
+if ( ! file_exists( $setup_file ) ) {
+  throw new Exception(
+    sprintf(
+      'setup.json is missing at this path: %s.',
+      $setup_file
+    )
+  );
+}
+
+// Parse json file to array.
+$data = json_decode( implode( ' ', (array) file( $setup_file ) ), true );
```

Comment:
```suggestion
$data = json_decode( file_get_contents( $setup_file ), true );
```

`file` returns an array, while `file_get_contents` returns a JSON string that can be decoded to array right away.

Reply:
We avoid using file_get_contents in WP so I haven't used here. But this is not WP specific so it should be ok. Tnx